### ISSUE TITLE
Add usbip as a preinstalled package

### ISSUE THREAD

**Issue by Managor** [ValveSoftware/SteamOS#1434]
#### Your system information

* Steam client version: 1710786209
* SteamOS version: 3.5.17
* Opted into Steam client beta?: Yes
* Opted into SteamOS beta?: Yes
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
I've seen many people attempt to use the steam deck as a remote controller without resorting to steam remote play. This could be handled using usbip. The most common commands can be found on the usbip tldr page https://github.com/tldr-pages/tldr/blob/main/pages/linux/usbip.md

I previously requested this on the Steam Forums: https://steamcommunity.com/app/1675200/discussions/2/7529517132616990830/

**Comment by qoh** [comment]
I agree that it would be preferable if it was just preinstalled, but it can still be done without any OS file changes/without enabling steamos-devmode.

One-time, download the package's files somewhere within your user folder:
```bash
mkdir ~/usbip
cd ~/usbip
curl -L https://archlinux.org/packages/extra/x86_64/usbip/download/ | tar --zstd -xf -
```

Start the server, list bindable USB devices, bind some, etc.:
```bash
cd ~/usbip
sudo modprobe usbip_host
sudo LD_LIBRARY_PATH=$(realpath usr/lib) usr/bin/usbipd &

sudo LD_LIBRARY_PATH=$(realpath usr/lib) usr/bin/usbip list -l
sudo LD_LIBRARY_PATH=$(realpath usr/lib) usr/bin/usbip bind -b <busid>
```

Tested on Steam Machine with a connected TM steering wheel, using it in a game streamed from a different PC.

**Comment by manuelafm** [comment]
Thanks for the suggestion.  This was added as pre-installed package for 3.10, will be in Main channel images soon.

If you have the chance to test it, please report back to tell if it works for you or if you find any problem.

**Comment by Managor** [comment]
Works great. Only minor issue I have with it is that if the deck ever goes to sleep, it detaches the usbip connection.
